docs/plates/_tpl_specimens.html

```
@@ -0,0 +1,487 @@
+<title>One Head, One Specimen</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)